### PR TITLE
MOTOR-1100 Update libmongocrypt payloads to new QE protocol

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

**Comment by blink1073** [comment]
Failures are pre-existing and tracked in https://jira.mongodb.org/browse/MOTOR-1133

File: tox.ini [hunk 1]
```diff
@@ -110,7 +110,8 @@ commands =
 
 [testenv:py3-pymongo-latest]
 commands =
-    pip install git+https://github.com/mongodb/mongo-python-driver.git@master#egg=pymongo[encryption]
+    pip install git+https://github.com/mongodb/mongo-python-driver.git@master
+    pip install --pre pymongocrypt
```
Comment (ShaneHarvey) [inline tox.ini:114]: I'm curious, why did you decide to use `--pre` even though we didn't publish a prerelease?

Comment (blink1073) [reply tox.ini:114]: I had mentioned in the ticket that this would allow us to publish a beta and consume it here.

Comment (blink1073) [reply tox.ini:114]: (in the future)

Comment (ShaneHarvey) [reply tox.ini:114]: SGTM

**Review by ShaneHarvey — APPROVED** [review]

**Pull request merged**